packages/interaction-context/src/InteractionContext.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { createContext, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useEventListener } from '@leafygreen-ui/hooks';
+
+export const UsingKeyboardContext = createContext(undefined as
+  | boolean
+  | undefined);
+
+interface InteractionContextProps {
+  children: React.ReactNode;
+  initialStates?: {
+    usingKeyboard?: boolean;
+  };
+}
+
+const InteractionContext = ({
```

Comment:
This should be `InteractionProvider`. The context contains the provider and the consumer.

Reply:
Fair enough!

Reply:
Changed the component to `LeafyGreenProvider`